Kd GVS bulk ingest docs (#8397)
broadinstitute/gatk: scripts/variantstore/beta_docs/run-your-own-samples.md

| --- | --- | --- |
| Ref | Reference allele. | --- |
| Alt | Alternate allele. | --- |
| AS_RAW_MQ, RAW_MQandDP, or RAW_MQ | RMS mapping quality (‘AS’: allele-specific). | Required for VQSR |
| AS_RAW_MQRankSum or Map_QUAL_RANK_SUM_KEY | Z-score from Wilcoxon rank sum test of alternate versus reference read mapping qualities. | Required for VQSR |
| QUALapprox | Sum of PL[0] values; used to approximate the QUAL score. | Required for VQSR |
| AS_QUALapprox | Allele-specific sum of PL[0] values; used to approximate the QUAL score. | Required for VQSR |
| AS_SB_TABLE or STRAND_BIAS_BY_SAMPLE | Allele-specific forward/reverse read counts for strand bias tests. | Required for VQSR |
| AS_VarDP, VarDP, or DP | Depth over variant genotypes, or read depth  (‘AS’: allele-specific). | Required for VQSR |
| call_GT | Genotype. | --- |
| call_GQ | Genotype quality. | --- |

### What does it return as output?

> The following files are stored in the workspace workflow execution bucket under Data>Files, or the google bucket specified in the inputs.

Reviewer: ```suggestion
The following files are stored in the workspace workflow execution bucket under Data>Files, or the Google bucket specified in the inputs.
```

> The following files are stored in the workspace workflow execution bucket under Data>Files, or the Google bucket specified in the inputs.

Reviewer 2: ```suggestion
The following files are stored in the workspace workflow execution bucket under Data>Files, or the Google bucket specified in the `extract_output_gcs_dir` input.
```

### Download the sample table template

The GVS workflow relies on the table structure and names in the Beta workspace. Download a TSV of the `sample` table to use as a template for uploading your own data.

1. Navigate to the **Data tab** in your clone of the GVS workspace.
2. Click on the **three vertical dots icon** next to the **sample** data table inside the TABLES sidebar on the left side of the page.
3. Select **Download TSV** to download the data table to your local machine.

### Delete the example data
You will need to delete the example sample data from your workspace so that it is not included in your callset.

1. Navigate to the **Data tab** in your clone of the GVS workspace.
2. Click on the **sample** data table. 
3. Click the **top check mark** to select all samples.
4. Click **Edit** and click **Delete selected rows** to delete the data.

### Edit TSV to upload data stored in the cloud

> If your data is already stored in the cloud, you’ll need to upload a TSV file to Terra containing the cloud paths to your files and update permissions on the data to allow Terra to access it. 

Reviewer 2: ```suggestion
If your data is already stored in the cloud, you’ll need to upload a TSV file to Terra containing the cloud paths to your files and update the cloud permissions on the data to allow your Terra proxy group to access it. 
```

1. **Open the TSV file** with a spreadsheet editor of your choice.
2. **Replace the cloud paths** to the example GVCF and index files in the second and third columns with the cloud paths to your GVCF and index files.

> 3. **Update the `sample_id` field** for each sample to anything you’d like. These will be used to name the samples in the output joint VCF file.

Reviewer 2: ```suggestion
3. **Update the `sample_id` field** for each sample to anything you’d like. These will be used to name the samples in the output joint VCF files.
```

---
4. Follow the steps in [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411) to **upload your data and TSV file** to Terra.

## Run the workflow

Now that your samples are loaded into data table in Terra, it’s time to setup and run the GVS workflow!

1. **Select the workflow** from the Workflows tab.
1. Configure the workflow inputs.
    1. Enter a **name for the callset** as a string with the format “*CALLSET_NAME*” for the `call_set_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable.
    1. Enter the name of the **GCP project** that holds the BigQuery dataset as a string with the format “*PROJECT_NAME*” for the `project_id` variable.

>     2. Enter the name of a **directory for writing outputs**. Enter a string in the format "*gs://your_bucket/here*" in `extract_output_gcs_dir`. If you want the data in your Terra workspace bucket you can find that on the Workspace Dashboard, right panel, under Cloud Information. Copy the "Bucket Name" and use it to the inputs to create a gs path as a string like this "*gs://fc-338fe040-3522-484c-ba48-14b48f9950d2*". If you do not enter a bucket here, the outputs will be in the execution directory under *Files*.

Reviewer 2: ```suggestion
    2. Enter the path of a **Google Cloud Storage directory for writing outputs**. Enter a string in the format "*gs://your_bucket/here*" in `extract_output_gcs_dir`. If you want the data in your Terra workspace bucket you can find that on the Workspace Dashboard, right panel, under Cloud Information. Copy the "Bucket Name" and use it to the inputs to create a gs path as a string like this "*gs://fc-338fe040-3522-484c-ba48-14b48f9950d2*". If you do not enter a bucket here, the outputs will be in the execution directory under *Files*.
```
